### PR TITLE
New feature/0.1/products

### PR DESCRIPTION
**Pull request by VladyslavBanul**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202862954456638
  - https://app.asana.com/0/0/1202862954456640

### CODE REVIEW

File: src/components/Table/styles.ts [hunk 1]
```diff
@@ -144,7 +144,9 @@ export const useStyles = makeStyles((theme: ThemeDefaultOptions) => ({
   infiniteScrollTableWrapper: {
     position: "relative",
     overflow: "auto",
-    height: "55vh",
-    maxHeight: 1000,
+    maxHeight: "55vh",
+    "@media screen and (min-height:1818px)": {
```
Comment (SergeyKoliadintsev) [inline src/components/Table/styles.ts:148]: image
We have breakpoint in our project, please try to implement this and remove @media

Comment (VladyslavBanul) [reply src/components/Table/styles.ts:148]: These breakpoints don't pass to this situation

File: src/levels/admin/components/Products/AddProduct/Steps/ProductDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        name: string().required(),
+      }),
+    ),
+  });
+
+  const { goNext, setPreviousChoices } = MultiStepForm;
+
+  useEffect(() => {
+    return () => {};
+  }, []);
```
Comment (SergeyKoliadintsev) [inline src/levels/admin/components/Products/AddProduct/Steps/ProductDetails.tsx:35]: You  need this useEffect ?

**Comment by SergeyKoliadintsev** [comment]
image
Add loader please

**Comment by SergeyKoliadintsev** [comment]
image

**Pull request merged**